### DISCUSSION TITLE
Darse de alta en Azure

### DISCUSSION THREAD

**Discussion by devcatz** [Launch-X-Latam/MisionFrontEnd#14] [Q&A]
Es necesario ya darse de alta en azure o solo cuando estemos en previos al modulo de azure?

**Comment by OscarTinajero117** [comment]
Todo con calma amig@

**Comment by antovanFI** [comment]
Aún no se ha dicho nada de darse de alta en Azure, cuando lleguemos al módulo nos darán las indicaciones

**Comment by doguedogue** [comment]
Aca algunos videos de innovaccion al respecto, aunque no han dicho nada sobre Azure hoy

[https://www.youtube.com/results?search_query=azure+innovaccion](https://www.youtube.com/results?search_query=azure+innovaccion)

**Comment by EddyMonters** [comment]
Todo a su tiempo, en el live del lunes, el Sherpa menciona que es importante ir leyendo, y que el contenido se irá desbloqueando conforme avance las siguientes semanas, por ahora respira profundo y a leer la primera parte de la sección, Saludos

**Comment by RafaelMartinCallataFlores** [comment]
Todavía no dijeron nada de dar alta en Azure seria bueno avanzar los temas con las sesiones
